src/js/components/visualization/GroupSubMenu.jsx

```diff
@@ -72,12 +73,16 @@ var Group = connect(
     return iconMarkup;
   },
   deleteUpdate: function(id) {
+    ReactTooltip.hide();
     this.deleteMark(id);
     // set selected to null
     this.props.select(null);
     // redraw sidebar
     this.updateSidebar();
   },
+  componentWillUnmount: function(){
+    ReactTooltip.hide();
+  },
```

Comment:
No because the inner marks are created through a loop that makes the HTML. So when the loop is modified it doesn't actually unmount this component.